src/aof.c

```diff
@@ -451,13 +457,20 @@ sds getNewBaseFileNameAndMarkPreAsHistory(aofManifest *am) {
  * for example:
  *  appendonly.aof.1.incr.aof
  */
-sds getNewIncrAofName(aofManifest *am) {
+sds getNewIncrAofName(aofManifest *am, int temp) {
```

Comment:
i think we must document what `temp` means (probably also also in the struct in the header`, and anywhere else we use that term for a function or variable name).
the word "temp" doesn't really explains it, and it's too common in software...
i.e. `An incremental file that only corresponds to a base file that's not yet created`

alternatively, maybe we should find a better term, one that is unique and then we can document it only once.
maybe "pending", or "standby"?

Reply:
No longer needed